app/webpacker/components/Panel/pages/RunValidatorsPage/ValidationListView.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useMemo } from 'react';
+import _ from 'lodash';
+import { Header, List, ListItem } from 'semantic-ui-react';
+import I18n from '../../../../lib/i18n';
+import { competitionUrl } from '../../../../lib/requests/routes.js.erb';
+
+const headingPrefixForType = (type) => {
+  switch (type) {
+    case 'info': return 'Information for';
+    case 'warning': return 'Warnings detected in';
+    case 'error': return 'Errors detected in';
+    default: return 'Unknown detected in';
+  }
+};
+
+export default function ValidationListView({ validations, showCompetitionNameOnOutput, type }) {
+  const listByGroup = useMemo(() => _.groupBy(validations, 'kind'), [validations]);
+
+  return (
+    <>
+      {Object.entries(listByGroup).map(([group, list]) => (
+        <>
```

Comment:
I'm pretty sure the console will be warning you that you need a key here.

Reply:
The console was filled with warnings and errors nowadays, so I'm not looking at it nowadays. Thanks for pointing it, I've made the change.